core/src/main/java/com/amplifyframework/hub/HubCategory.java

```diff
@@ -72,33 +70,6 @@ public CategoryType getCategoryType() {
         return CategoryType.HUB;
     }
 
-    /**
-     * Convenience method to allow callers to subscribe to Hub events
-     * for a particular operation.  Internally, the subscription
-     * transforms the HubEvent into the Operation's expected AsyncEvent
-     * type, so callers may re-use their `subscriber`s.
-     *
-     * @param operation The operation to subscribe to events for
-     * @param eventListener The Operation-specific listener to be invoked
-     *                 when an AsyncEvent for that operation is received.
-     * @param <E> The type of the event that the event listener will receive
-     * @param <R> The type of the request object of the {@link AmplifyOperation}
-     * @return A subscription token
-     */
-    @NonNull
-    public <E, R> SubscriptionToken subscribe(
-            @NonNull final AmplifyOperation<R> operation,
-            @SuppressWarnings("unused") @NonNull final EventListener<E> eventListener) {
-        HubChannel channel = HubChannel.forCategoryType(operation.getCategoryType());
-        HubSubscriber transformingListener = event -> {
-            // TODO: check for casting of Object to E and
-            // see if it can be prevented.
-            // eventListener.onEvent(event.getData());
-        };
-
-        return subscribe(channel, event -> true, transformingListener);
-    }
-
```

Comment:
Ah I see you are removing this also. Nevermind about my comment about operation ID then. 

Would it be a non-trivial task to fix this before GA?

Reply:
I haven't thought much about what we would need to do here. I think it will take a bit of study to see what iOS is doing, and rebuild from that. We should talk to Tim, and revisit this Hub integration, probably after GA.